From ac3d490aaee0ec359e1b65fd91ddcc9991bde94c Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 25 Oct 2012 11:23:48 +0800
Subject: [PATCH 085/602] ENGR00230038 sabresd: Add xlarge screen size to aapt
 config

LVDS screen for SabreSD has a xlarge screen size up to
10'. So we must let aapt tools to build in all the
xlarge layouts and res into apk.
Otherwise the settings fragment label will disappear,
the livewallpaper selector will shows like phone UI,
the gallery2 will only have 2x4 layouts with huge thumbnail.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/sabresd/SabreSDProductCommon.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/sabresd/SabreSDProductCommon.mk b/imx6/sabresd/SabreSDProductCommon.mk
index fb665f9..805f07c 100644
--- a/imx6/sabresd/SabreSDProductCommon.mk
+++ b/imx6/sabresd/SabreSDProductCommon.mk
@@ -10,7 +10,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/imx6/sabresd/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/tablet_core_hardware.xml:system/etc/permissions/tablet_core_hardware.xml \
-- 
1.8.0

